faq.md

```diff
@@ -73,11 +73,11 @@ A version of the spec is available in the docs repo in this project
 ## Is there a cost for joining? 
 There is no cost for anyone to join and contribute. 
 
-## Do I need to be Linux Foundation member to join?  
+## Do I need to be a Linux Foundation member to join?  
 Anyone can join and contribute. You don’t need to be a Linux Foundation member. 
 
-## Where can I report a bug?
-Vulnerability reports can be sent to info@opea.dev.
+## Where can I report a security vulnerability?
+Vulnerability reports can be sent to security@opea.dev.
```

Comment:
```suggestion
Send vulnerability reports to security@opea.dev.
```
I suspect we'd want to include what information they should include in that report email?

Reply:
Typically, we provide a template for all issues and perhaps a different one for security issues. We could follow the patterns on Kubernetes, or other project.

Reply:
But we don't want to create a GitHub issue for security concerns. Such reporting needs to be done privately.